src/Error/ExceptionTrap.php

```diff
@@ -169,6 +169,8 @@ public function register(): void
         set_exception_handler($this->handleException(...));
         register_shutdown_function($this->handleShutdown(...));
         static::$registeredTrap = $this;
+
+        ini_set('assert.exception', '1');
```

Comment:
I hope this is the right place to set this ini config since hte exception handler is registered through app's bootstrap.